test/mpi/dtpools/include/dtpools_internal.h

```diff
@@ -121,9 +121,94 @@ typedef enum {
     DTPI_OBJ_TYPE__SUBARRAY_C,
     DTPI_OBJ_TYPE__SUBARRAY_F,
     DTPI_OBJ_TYPE__STRUCT,
+/*
```

Comment:
The patch "test/dtpools: refactoring of object free and check functions":

It would be nice to say what was the goal of the refactoring (code movement? naming change? what else?)

Reply:
I will add an explanation to the commit message. The main goal was to have for struct pools the same type of behavior of basic pools; that is, you can build derived datatypes using an arbitrary struct instead of a predefined type. I have that implemented in a separate branch but I need to do some fix up.